src/auth/views/login.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Formik } from 'formik';
+import { Link } from 'react-router-dom';
+import Input from 'common/components/input';
+import { adminLogin } from 'api/request.api';
+import SubmitButton from 'common/components';
+import { LoginPayload } from 'auth/auth.types';
+import { useTranslation } from 'react-i18next';
+import { AuthLayout } from 'layouts/auth-layout';
+import { useModal, Modal } from 'common/components/modal';
+import { loginValidationSchema } from 'auth/auth.validation';
+
+const Login = () => {
+  const { t } = useTranslation();
+  const loginModal = useModal(true);
+
+  const initialValues: LoginPayload = {
+    email: '',
+    password: '',
+  };
```

Comment:
These kinda variables can be placed outside the Component because every time Login re-renders a new object is created and this initial value has nothing to do with this because we never gonna update their values .